### PR TITLE
Improve batching in metrics and traces published

### PR DESCRIPTION
**Pull request by nadundesilva**

### CODE REVIEW

File: misc/tracing-extensions/modules/ballerina-choreo-extension/src/main/java/org/ballerinalang/observe/trace/extension/choreo/ChoreoJaegerReporter.java [hunk 1]
```diff
@@ -33,58 +33,64 @@
...
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.ScheduledThreadPoolExecutor;
+import java.util.concurrent.TimeUnit;
 
 /**
  * Custom Jaeger tracing reporter for publishing stats to Choreo cloud.
  */
 public class ChoreoJaegerReporter implements Reporter, AutoCloseable {
-    private static final int PUBLISH_INTERVAL = 10 * 1000;
+    private static final int PUBLISH_INTERVAL = 10;
```
Comment (afkham) [inline misc/tracing-extensions/modules/ballerina-choreo-extension/src/main/java/org/ballerinalang/observe/trace/extension/choreo/ChoreoJaegerReporter.java:44]: PUBLISH_INTERVAL_MINS ?

File: misc/tracing-extensions/modules/ballerina-choreo-extension/src/main/java/org/ballerinalang/observe/trace/extension/choreo/MetricsReporterExtension.java [hunk 2]
```diff
@@ -50,7 +52,7 @@
 public class MetricsReporterExtension implements MetricReporter, AutoCloseable {
     private static final Logger LOGGER = LogFactory.getLogger();
 
-    private static final int PUBLISH_INTERVAL = 10 * 1000;
+    private static final int PUBLISH_INTERVAL = 10;
```
Comment (afkham) [inline misc/tracing-extensions/modules/ballerina-choreo-extension/src/main/java/org/ballerinalang/observe/trace/extension/choreo/MetricsReporterExtension.java:55]: PUBLISH_INTERVAL_SECS ?

File: misc/tracing-extensions/modules/ballerina-choreo-extension/src/main/java/org/ballerinalang/observe/trace/extension/choreo/ChoreoJaegerReporter.java (hunk 1 above)
Comment (nadundesilva) [reply misc/tracing-extensions/modules/ballerina-choreo-extension/src/main/java/org/ballerinalang/observe/trace/extension/choreo/ChoreoJaegerReporter.java]: Fixed with b2e6558

File: misc/tracing-extensions/modules/ballerina-choreo-extension/src/main/java/org/ballerinalang/observe/trace/extension/choreo/MetricsReporterExtension.java (hunk 2 above)
Comment (nadundesilva) [reply misc/tracing-extensions/modules/ballerina-choreo-extension/src/main/java/org/ballerinalang/observe/trace/extension/choreo/MetricsReporterExtension.java]: Fixed with b2e6558

**Pull request merged**